lib/internal/worker.js

```diff
@@ -80,6 +80,15 @@ class Worker extends EventEmitter {
                                      'array',
                                      options.execArgv);
     }
+    let argv;
+    if (options.argv) {
+      if (!Array.isArray(options.argv)) {
+        throw new ERR_INVALID_ARG_TYPE('options.argv',
+                                       'array',
```

Comment:
Not sure but I think we use
```suggestion
                                       'Array',
```

Reply:
I've found mixed usages on whether or not capitalizing the name. Is there a rule on it?

Reply:
Well, the only lowercase I have found is in the `internal/worker.js` so that looks like a typo to me.